nflow-tests/src/test/java/io/nflow/tests/CreditApplicationWorkflowTest.java

```diff
@@ -94,14 +94,14 @@ public void checkErrorStateReached() {
   public void checkWorkflowInstanceActions() {
     int i = 1;
     assertWorkflowInstance(resp.id, actionHistoryValidator(asList(
-        new Action(i++, stateExecution.name(), "error", "", 0, null, null, 0),
-        new Action(i++, stateExecution.name(), "grantLoan", "", 3, null, null, 0),
-        new Action(i++, stateExecution.name(), "grantLoan", "", 2, null, null, 0),
-        new Action(i++, stateExecution.name(), "grantLoan", "", 1, null, null, 0),
-        new Action(i++, stateExecution.name(), "grantLoan", "", 0, null, null, 0),
-        new Action(i++, stateExecution.name(), "grantLoan", "", 0, null, null, 0),
-        new Action(i++, stateExecution.name(), "acceptCreditApplication", "", 0, null, null, 0), // probably not the way to show manual action in future
-        new Action(i++, stateExecution.name(), "createCreditApplication", "", 0, null, null, 0))));
+        new Action(0, stateExecution.name(), "error", "", 0, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 3, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 2, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 1, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 0, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 0, null, null, 0),
+        new Action(0, stateExecution.name(), "acceptCreditApplication", "", 0, null, null, 0), // probably not the way to show manual action in future
+        new Action(0, stateExecution.name(), "createCreditApplication", "", 0, null, null, 0))));
```

Comment:
Why this change?

Reply:
it was unused. So let's initialize it to null/zero value like the other fields in the actions.
Noticed it because I had one random failure in the tests and had  spared a glance at the code.
I can revert this change from the PR since it is not needed for the version bump

Reply:
Ok to keep this then